Add GH Action for Publishing eCTD bundle to repository

rpodcast opened the pull request:
This PR adds the necessary GitHub action workflow and script for creating the eCTD bundle structure dynamically and then pushing the changes to the upstream Pilot 5 submission bundle repository. Note that the steps associated with copying the cover letter and readme of the repository are disabled until we have initial versions of those files in the repository.

elimillera commented on .github/workflows/publish-ectd-bundle.yaml:
```diff
@@ -0,0 +1,93 @@
+            path: ./submissions-pilot5-datasetjson-to-fda
+            token: ${{ secrets.API_TOKEN_GITHUB }}
+  
+        - name: Create ECTD bundle
+          run: bash ./.github/scripts/create-ectd-bundle.sh
+  
```
Along with creating the ectd-bundle, I think it would be helpful to add it as an artifact in the pipeline to be able to link it to a release and troubleshoot if anything goes wrong: https://docs.github.com/en/actions/tutorials/store-and-share-data

rpodcast commented:
@elimillera I've addressed your comment with a new step in the action file, so I think we are good to go and I'll get this merged now since I will need this for my subsequent updates. Thank you for your review!

bms63 commented:
@rpodcast this is showing up? 
<img width="1139" height="611" alt="image" src="https://github.com/user-attachments/assets/510a9137-90c7-4b89-b467-03c44d2a111e" />
